package.json

```diff
@@ -68,84 +68,84 @@
-    "@bbc/psammead-locales": "^5.0.0",
-    "@bbc/psammead-media-indicator": "^6.0.7",
-    "@bbc/psammead-media-player": "^5.0.11",
-    "@bbc/psammead-most-read": "^6.0.11",
-    "@bbc/psammead-navigation": "^9.0.1",
-    "@bbc/psammead-paragraph": "^4.0.5",
+    "@bbc/psammead-calendars": "2.0.19",
+    "@bbc/psammead-caption": "4.1.2",
+    "@bbc/psammead-consent-banner": "4.0.6",
+    "@bbc/psammead-copyright": "3.0.6",
+    "@bbc/psammead-detokeniser": "1.0.0",
+    "@bbc/psammead-embed-error": "3.0.8",
+    "@bbc/psammead-episode-list": "1.0.5",
+    "@bbc/psammead-figure": "2.0.1",
+    "@bbc/psammead-grid": "3.0.9",
+    "@bbc/psammead-heading-index": "3.0.6",
+    "@bbc/psammead-headings": "5.0.6",
+    "@bbc/psammead-image": "2.0.1",
+    "@bbc/psammead-image-placeholder": "3.0.8",
+    "@bbc/psammead-inline-link": "3.0.5",
+    "@bbc/psammead-live-label": "2.0.7",
+    "@bbc/psammead-locales": "5.0.0",
+    "@bbc/psammead-media-indicator": "6.0.8",
+    "@bbc/psammead-media-player": "5.0.15",
+    "@bbc/psammead-most-read": "6.0.14",
+    "@bbc/psammead-navigation": "9.0.1",
+    "@bbc/psammead-paragraph": "4.0.6",
     "@bbc/psammead-podcast-promo": "0.1.0-alpha.24",
-    "@bbc/psammead-radio-schedule": "^5.0.16",
-    "@bbc/psammead-rich-text-transforms": "^2.0.2",
-    "@bbc/psammead-script-link": "^3.0.6",
-    "@bbc/psammead-section-label": "^7.0.6",
-    "@bbc/psammead-sitewide-links": "^6.0.5",
-    "@bbc/psammead-social-embed": "^3.1.2",
-    "@bbc/psammead-story-promo": "^8.0.7",
-    "@bbc/psammead-story-promo-list": "^6.0.5",
-    "@bbc/psammead-styles": "^7.0.3",
-    "@bbc/psammead-timestamp-container": "^5.0.15",
-    "@bbc/psammead-useful-links": "^3.0.6",
-    "@bbc/psammead-visually-hidden-text": "^2.0.1",
-    "@bbc/web-vitals": "^1.0.2",
-    "@emotion/cache": "^10.0.29",
-    "@emotion/core": "^10.0.35",
-    "@emotion/styled": "^10.0.27",
-    "@loadable/component": "~5.14.0",
-    "@loadable/server": "~5.14.0",
-    "aws-embedded-metrics": "^2.0.0",
-    "compression": "^1.7.4",
-    "core-js": "^3.6.5",
-    "create-emotion-server": "^10.0.27",
-    "dotenv": "^8.2.0",
-    "express": "^4.17.1",
-    "express-static-gzip": "^2.0.8",
+    "@bbc/psammead-radio-schedule": "5.1.7",
+    "@bbc/psammead-rich-text-transforms": "2.0.2",
+    "@bbc/psammead-script-link": "3.0.7",
+    "@bbc/psammead-section-label": "7.0.7",
+    "@bbc/psammead-sitewide-links": "6.0.6",
+    "@bbc/psammead-social-embed": "3.1.3",
+    "@bbc/psammead-story-promo": "8.0.8",
+    "@bbc/psammead-story-promo-list": "6.0.6",
+    "@bbc/psammead-styles": "7.0.3",
+    "@bbc/psammead-timestamp-container": "5.0.16",
+    "@bbc/psammead-useful-links": "3.0.7",
+    "@bbc/psammead-visually-hidden-text": "2.0.1",
+    "@bbc/web-vitals": "1.0.2",
+    "@emotion/cache": "10.0.29",
+    "@emotion/core": "10.1.1",
+    "@emotion/styled": "10.0.27",
+    "@loadable/component": "5.14.1",
+    "@loadable/server": "5.14.0",
```

Comment:
I'd make sure these 2 are on the same version

Reply:
This must be what our current lock file uses, not sure I should change that in this PR?

Reply:
Yeh no worries then. My Yarn migration PR will take care of it 🙂